R/smoking.R

```diff
@@ -0,0 +1,60 @@
+#' @title Smoking pack-years
+#' 
+#' @details Measures an individual smoking pack-years based on various CCHS smoking variables
```

Comment:
More description on what pack-years is in case they are not familiar with the term

Reply:
Agree. a definition or example of how to make a calculation would be helpful. 
"`pack-years` is calculated by multiplying the number of cigarette packs per day (20 cigarettes per pack) by the number of years. Example 1: a respondent who is a current smoker who smokes 1 package of cigarettes for the last 10 years has smoked 10 pack-years. `pack-years` is also calculated for former smokers. Example 2: a respondent who started smoking at age 20 years and smoked half a pack of cigarettes until age 40 years smoked for 10 pack-years. 
?note about occasional smokers?